test/scripts/sports_data_from_sportradar.txtar

```diff
@@ -0,0 +1,8 @@
+# [execute] attest fn call to SportRadar API
+stdin fn-call.json
+exec bky-as attest-fn-call
+cp stdout result.json
+
+# [check] assert stdout contains correct game information
+exec jq -c -r '.transitive_attested_function_call.claims.output | @base64d | fromjson' result.json
+stdout '{"Success":true,"Error":"","Value":{"points_per_minute":'
```

Comment:
```suggestion
exec jq -r '.transitive_attested_function_call.claims.output | @base64d | fromjson' result.json
cmp stdout expected-output.json

-- expected-output.json --
{
  "Success": true,
  "Error": "",
  "Value": {
    "points_per_minute": [
      {
        "player": "Jayson Tatum",
        "ppm": 0.689655172413793
      },
      {
        "player": "Luka Doncic",
        "ppm": 0.6454091432961967
      }
    ],
    "winner": "Jayson Tatum"
  }
}
```

Reply:
made this change myself while updating other files